db/migrate/20201023133523_create_calculated_handover_dates.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCalculatedHandoverDates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :calculated_handover_dates do |t|
+      t.string :nomis_offender_id, null: false
```

Comment:
it would be nice to maybe use rails conventions here - use the case information id rather than nomis_offender_id everywhere.

Reply:
@starswan I've changed this to use `references`, which also adds a foreign key.